beetsplug/discogs.py

```diff
@@ -434,9 +434,12 @@ def get_album_info(self, result):
 
         # Extract information for the optional AlbumInfo fields that are
         # contained on nested discogs fields.
-        media, albumtype = self.get_media_and_albumtype(
-            result.data.get("formats")
-        )
+        albumtypes = albumtype = media = label = catalogno = labelid = None
+        if result.data.get("formats"):
+            albumtypes = result.data["formats"][0].get("descriptions", [])
+            if len(albumtypes) > 0:
+                albumtype = albumtypes[0]
+            media = result.data["formats"][0]["name"]
```

Comment:
Sorry that is clumsy code and I didn't look into the helper function someone provided (`get_media_and_albumtype()`) yet, but you get the point. Now a list field `albumtypes` is existing and yes I make it easy for `albumtype` being fixed here already. 

Actually setting albumtypes to an empty list here would have been sufficient, right?

Reply:
As you describe below. That's not it.... :-/

Reply:
See my comment here: https://github.com/beetbox/beets/pull/5540#discussion_r1885384710. `albumtypes` _does_ get set to an empty list even when it's not provided